eZ/Publish/Core/Persistence/Legacy/User/Handler.php

```diff
@@ -512,12 +512,40 @@ public function assignRole($contentId, $roleId, array $limitation = null)
      * @param mixed $contentId The user or user group Id to un-assign the role from.
      * @param mixed $roleId
      */
-    public function unAssignRole($contentId, $roleId)
+    public function unassignRole($contentId, $roleId)
     {
         $this->userGateway->removeRole($contentId, $roleId);
     }
 
     /**
+     * Un-assign a role by assignment ID.
+     *
+     * @param mixed $roleAssignmentId The assignment ID.
+     */
+    public function removeRoleAssignment($roleAssignmentId)
+    {
+        $this->userGateway->removeRoleAssignmentById($roleAssignmentId);
+    }
+
+    /**
+     * Loads role assignment for specified assignment ID.
+     *
+     * @param mixed $roleAssignmentId
+     *
+     * @return \eZ\Publish\SPI\Persistence\User\RoleAssignment
+     */
+    public function loadRoleAssignment($roleAssignmentId)
+    {
+        $data = $this->roleGateway->loadRoleAssignment($roleAssignmentId);
+
+        if (empty($data)) {
+            return array();
```

Comment:
load* methods in SPI should typically either return the object or throw not found exception.